test/metabase/lib/schema/expression_test.cljc

```diff
@@ -0,0 +1,27 @@
+(ns metabase.lib.schema.expression-test
+  (:require
+   [clojure.test :refer [are deftest is testing]]
+   [malli.core :as mc]
+   [metabase.lib.schema.expression :as expression]
+   [metabase.lib.test-metadata :as meta]))
+
+(deftest ^:parallel integer-expression-test
```

Comment:
Since the schemas are getting hairy, I added some tests to make sure they actually did what I thought they were doing